fuzzing/extract_text_fuzzer.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+import atheris
+
+from fuzz_helpers import EnhancedFuzzedDataProvider
```

Comment:
Would it be possible to start all imports from the root of the project? Usually that makes it easier to get the imports working. 

So that would require `from fuzzing.fuzz_helpers import ...` here. 

Add a `__init__.py` file to the `fuzzing` dir should get it to work.